elastic4s-core/src/main/scala/com/sksamuel/elastic4s/update/UpdateByQueryDefinition.scala

```diff
@@ -30,6 +31,8 @@ case class UpdateByQueryDefinition(sourceIndexes: Indexes,
 
   def refresh(refresh: Boolean): UpdateByQueryDefinition = copy(refresh = refresh.some)
```

Comment:
I noticed that DeleteByQueryDefinition uses a RefreshPolicy Enum for this, and now sets the query parameter based on it. Would you like me to make the equivalent change here?

Reply:
Please.